AmplifyPlugins/API/AWSAPICategoryPluginFunctionalTests/GraphQLModelBased/GraphQLConnectionScenario3Tests.swift

```diff
@@ -53,6 +53,37 @@ class GraphQLConnectionScenario3Tests: XCTestCase {
         Amplify.reset()
     }
 
+    func testQuerySinglePost() {
+        guard let post = createPost(title: "title") else {
+            XCTFail("Failed to set up test")
+            return
+        }
+
+        let completeInvoked = expectation(description: "request completed")
```

Comment:
s/completeInvoked/queryInvokedSuccessfully/

Reply:
replacing with requestInvokedSuccessfully as it is easiest since a lot of helper code that uses this